### DISCUSSION TITLE
Capture above comment placeholder

### DISCUSSION THREAD

**Discussion by dpw67** [chhoumann/quickadd#532] [Q&A]
Is there a way to capture a journal (or task) entry in my Daily Note before a one-line comment placeholder (like %% new-journal-here %%? Currently, I use Actions for Obsidian with the "Insert Text At Placeholder in Daily Note" action (with "before placeholder" selected for the option "Where should text be inserted?". That works fine with new journal entries always being added after previous one before my placeholder. 

Although I can use the Shortcut from my menu bar when I'm in Obsidian, I thought it would be nice to have a similar Quick Add command to do the same thing. But that doesn't seem possible. Since, there's no way in Actions for Obsidian to insert text at the beginning or end of a section like Quick Add), I have to choose which one to use since they're not compatible (and Shortcuts currently works everywhere, whereas Quick Add only within Obsidian).

When I try to use the Quick Add plugin to capture a journal (or task) entry, I can only insert the text at the beginning or end of my ## Journal section, so all the Quick Add entries appear after my placeholder (with all the Shortcut entries above). Then I have to manually move all the Quick Add entries above the placeholder, and adjust their order if the times are intermixed between the two.

I do not see any where way to have Quick Add insert the text before the placeholder. Is there some way I can do this?

**Comment by chhoumann** [comment]
Very late reply here - sorry about that.

Capture now has an **Insert before** placement (shipped in QuickAdd 2.13.0), which is exactly the "before placeholder" behavior you described from Actions for Obsidian. Point your Capture choice at the daily note, enable Insert before, and set the target line to:

```
%% new-journal-here %%
```

Each capture is inserted directly above the placeholder, so successive entries stack in order above it - no more manual reshuffling, and it stays compatible with entries your Shortcut adds the same way.

Two details that help here: the target line accepts format syntax, and **Create line if not found** bootstraps the placeholder in daily notes that don't have it yet (for Insert before, the captured content is written first and the placeholder line is created below it).

Docs: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice#insert-before

Closing as resolved.
